clang-ocl.in

```diff
@@ -64,7 +64,7 @@ ${CLANG} -c -emit-llvm \
 -Xclang -cl-ext=+cl_khr_fp64,+cl_khr_global_int32_base_atomics,+cl_khr_global_int32_extended_atomics,+cl_khr_local_int32_base_atomics,+cl_khr_local_int32_extended_atomics,+cl_khr_int64_base_atomics,+cl_khr_int64_extended_atomics,+cl_khr_3d_image_writes,+cl_khr_byte_addressable_store,+cl_khr_gl_sharing,+cl_amd_media_ops,+cl_amd_media_ops2,+cl_khr_subgroups \
 -include ${OPENCL_INCLUDE}/opencl-c.h \
 ${BITCODE_ARGS} \
-${options} -o ${output_file}.linked.bc ${input_file}
+${options} -o ${output_file}.linked.bc ${input_file} |& tee ${output_file}.linked.bc.out
```

Comment:
Will this include compiler warnings? Previously, this was grepping the warnings during the link stage only.

Reply:
I removed the `grep` completely in PR #27.

Reply:
Yes this would include all warnings including compiler warnings during the optimization and codegen phases. It won't be checking just link stage.

Reply:
> I removed the `grep` completely in PR #27.

Okay, maybe that is the better approach.